### PR TITLE
refactored output.py to correct vim behavior

### PR DESCRIPTION
**Pull request by brwong**
- changed vim output from columns to tabs
- changed vim command so that jump-to-line commands work

(Feel free to modify as needed.)

### CODE REVIEW

**Comment by brwong** [comment]
My vim command reference, FYI

[Stack Overflow - Vim : Open multiples files on different lines](http://stackoverflow.com/questions/2148397/vim-open-multiples-files-on-different-lines/2148692#2148692)

**Comment by pcottle** [comment]
Hrm, most people at work I talk to prefer columns rather than tabs (just because its immediately clear that `$x` files are open). And my other worry is that this has been the default behavior for over a year, and changing in this release would be a bit thrashy

would you be open to putting this behind some kind of system flag?

**Comment by brwong** [comment]
I changed it so that "vim" acts as before (but with the correct line number jumps), and setting FPP_EDITOR to "vim -p" opens new tabs.

As usual, I'm open to suggestions.

File: scripts/makeDist.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 VERSION="$(python ./src/version.py)"
 DEST="./dist/fpp.$VERSION.tar.gz"
 mkdir -p ./dist/
-tar -cf $DEST src/*.py fpp
+tar -czf $DEST src/*.py fpp
```
Comment (pcottle) [inline scripts/makeDist.sh]: this is `czf` in trunk and was changed recently. you probably reverted this in some kind of rebase or something -- mind dropping the change to this file?

File: src/output.py [hunk 2]
```diff
@@ -94,33 +92,36 @@ def getEditorAndPath():
...
+    if editor == 'vim -p':
+        firstFilePath, firstLineNum = filesAndLineNumbers[0]
+        cmd += ' +%d %s' % (firstLineNum, firstFilePath)
+        for (filePath, lineNum) in filesAndLineNumbers[1:]:
+            cmd += ' +"tabnew +%d %s"' % (lineNum, filePath)
+    elif editor == 'vim':
+        firstFilePath, firstLineNum = filesAndLineNumbers[0]
+        cmd += ' +%d %s' % (firstLineNum, firstFilePath)
+        for (filePath, lineNum) in filesAndLineNumbers[1:]:
+            cmd += ' +"vsp +%d %s"' % (lineNum, filePath)
```
Comment (pcottle) [inline src/output.py]: woahhh so `vsp` does vertical split like `-O` and allows you to specify the line numbers? very cool!

File: src/output.py [hunk 3]
```diff
@@ -94,33 +92,36 @@ def getEditorAndPath():
     return 'vim', 'vim'
 
 
-def getEditFileCommand(filePath, lineNum):
```
Comment (pcottle) [inline src/output.py]: ah i see why this had to go -- yeah you didnt have global context when composing these mini-parts.

File: src/output.py [hunk 4]
```diff
@@ -94,33 +92,36 @@ def getEditorAndPath():
...
-        return editor_path + ' -O ' + ' '.join(partialCommands)
-    # Assume that all other editors behave like emacs
-    return editor_path + ' ' + ' '.join(partialCommands)
+    cmd = editor_path + ' '
+    if editor == 'vim -p':
+        firstFilePath, firstLineNum = filesAndLineNumbers[0]
+        cmd += ' +%d %s' % (firstLineNum, firstFilePath)
+        for (filePath, lineNum) in filesAndLineNumbers[1:]:
+            cmd += ' +"tabnew +%d %s"' % (lineNum, filePath)
+    elif editor == 'vim':
```
Comment (pcottle) [inline src/output.py]: note to self -- we lost the `FPP_DISABLE_SPLIT` behavior here so lets add that back

**Pull request merged**